.github/workflows/ansible-lint.yml

```diff
@@ -0,0 +1,19 @@
+## Source: https://github.com/marketplace/actions/ansible-lint
+
+name: Ansible Lint # feel free to pick your own name
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Important: This sets up your GITHUB_WORKSPACE environment variable
+      - uses: actions/checkout@v2
+
+      - name: Run ansible-lint
```

Comment:
To have 5 star on ansible-galaxy it is also needed to run yamllint together with ansible-lint. One can do it with:

```md
      - name: Set up Python
        uses: actions/setup-python@v4
        with:
          python-version: "3.10"

      - name: Install test dependencies
        run: pip install --constraint=.github/workflows/constraints.txt ansible yamllint ansible-lint

      - name: Lint code
        run: |
          yamllint .
          ansible-lint
```

Note that versions of yamllint and ansible-lint are pinned so a constants.txt file in the same folder with:

```
pip==22.1.2
ansible==6.0.0
yamllint==1.26.3
ansible-lint==6.3.0
```